MAUI/ImageEditor/text.md

```diff
@@ -88,6 +89,38 @@ Customize the appearance of text annotation using the [`ImageEditorTextSettings`
 
 ![Text annotation customization in .NET Maui ImageEditor](images/text/imageeditor-text-customization.png)
 
+## Customize the background color of text annotations
+
+The background color of the text annotations can be customized using the `Background` property.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+   <Grid RowDefinitions="0.9*, 0.1*">
+        <imageEditor:SfImageEditor x:Name="imageEditor"
+                                   Source="image.jpeg" />
+        <Button Grid.Row="1"
+                Text="TextBackground"
+                Clicked="OnTextBackgroundClicked" />
+    </Grid>  
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+    private void OnTextBackgroundClicked(object sender, EventArgs e)
+    {
+        this.imageEditor.AddText("Good Day",
+           new ImageEditorTextSettings()
+           {
+               Background = Colors.White.WithAlpha(0.5f)
```

Comment:
Background = Colors.White.WithAlpha(0.5f)

no need to mention here, use like below

Background = Colors.White,
Opacity = 0.5

Reply:
Changed